src/uwtools/config/formats/fieldtable.py

```diff
@@ -52,37 +65,25 @@ def dump_dict(cls, cfg: dict, path: Optional[Path] = None) -> None:
         """
         Dumps a provided config dictionary in Field Table format.
 
-        FMS field and tracer managers must be registered in an ASCII table called 'field_table'.
+        FMS field and tracer managers must be registered in an ASCII table called ``field_table``.
```

Comment:
We may find that we need yet more RST syntax in docstrings that are now being rendered to RtD. I got the obvious ones, but be on the lookout when perusing the RtD docs for more cases where formatting could look nicer.